src/main/java/no/nav/familie/integrasjoner/kodeverk/KodeverkController.kt

```diff
@@ -38,4 +38,9 @@ class KodeverkController(val kodeverkService: KodeverkService) {
     fun hentLandkod(@PathVariable landkode: String): ResponseEntity<Ressurs<String>> {
         return ResponseEntity.ok(Ressurs.Companion.success(kodeverkService.hentLandkode(landkode)))
     }
+
+    @GetMapping("/landkoder/EEA")
```

Comment:
Navnekonventioner for rest url tror jeg er at alltid bruke små bokstaver (https://restfulapi.net/resource-naming/)